src/cli/config.rs

```diff
@@ -1,347 +0,0 @@
-                        return Ok(Some(config));
-                    }
-                }
-            }
-        }
-
-        Ok(None)
-    }
-}
-
-/// Searches the directory for the configuration toml file (i.e. `stylua.toml` or `.stylua.toml`)
-fn find_toml_file(directory: &Path) -> Option<PathBuf> {
-    for name in &CONFIG_FILE_NAME {
-        let file_path = directory.join(name);
-        if file_path.exists() {
-            return Some(file_path);
-        }
-    }
-
-    None
-}
-
-pub fn find_ignore_file_path(mut directory: PathBuf, recursive: bool) -> Option<PathBuf> {
-    debug!("config: looking for ignore file in {}", directory.display());
-    let file_path = directory.join(".styluaignore");
-    if file_path.is_file() {
-        debug!("config: resolved ignore file at {}", file_path.display());
-        Some(file_path)
-    } else if recursive && directory.pop() {
-        find_ignore_file_path(directory, recursive)
-    } else {
-        None
-    }
-}
-
-/// Handles any overrides provided by command line options
-fn load_overrides(config: Config, opt: &Opt) -> Config {
-    let mut new_config = config;
-
-    if let Some(syntax) = opt.format_opts.syntax {
-        new_config.syntax = syntax.into();
-    };
-    if let Some(column_width) = opt.format_opts.column_width {
-        new_config.column_width = column_width;
-    };
-    if let Some(line_endings) = opt.format_opts.line_endings {
-        new_config.line_endings = line_endings.into();
-    };
-    if let Some(indent_type) = opt.format_opts.indent_type {
-        new_config.indent_type = indent_type.into();
-    };
-    if let Some(indent_width) = opt.format_opts.indent_width {
-        new_config.indent_width = indent_width;
-    };
-    if let Some(quote_style) = opt.format_opts.quote_style {
-        new_config.quote_style = quote_style.into();
-    };
-    if let Some(call_parentheses) = opt.format_opts.call_parentheses {
-        new_config.call_parentheses = call_parentheses.into();
-    };
-    if let Some(space_after_function_names) = opt.format_opts.space_after_function_names {
-        new_config.space_after_function_names = space_after_function_names.into();
-    };
-    if let Some(collapse_simple_statement) = opt.format_opts.collapse_simple_statement {
-        new_config.collapse_simple_statement = collapse_simple_statement.into();
-    }
-    if opt.format_opts.sort_requires {
-        new_config.sort_requires = SortRequiresConfig { enabled: true }
-    }
-
-    new_config
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use clap::StructOpt;
-    use stylua_lib::{CallParenType, IndentType, LineEndings, LuaVersion, QuoteStyle};
-
-    #[test]
-    fn test_override_syntax() {
-        let override_opt = Opt::parse_from(vec!["BINARY_NAME", "--syntax", "Lua51"]);
-        let default_config = Config::new();
-        let config = load_overrides(default_config, &override_opt);
-        assert_eq!(config.syntax, LuaVersion::Lua51);
-    }
```

Comment:
Those tests are no longer possible, as the `load_overrides` was moved to the lib part. Testing this function there, without involving `Opt::parse_from` seemed pointless to me. So I removed them.